### PR TITLE
Chore: Set Accessibility Identifiers for Startup, Signin. Register, WhatsNew Screens

### PR DESCRIPTION
**Pull request by saeedbashir**
This PR sets accessibility identifiers for startup, signin, forgot password and whats new screens. These accessibility identifiers will be used by the QA team to automate the testing flow.

### CODE REVIEW

**Review by rnr — DISMISSED** [review]

**Review by volodymyr-chekyrta — DISMISSED** [review]

File: Authorization/Authorization/Presentation/Reset Password/ResetPasswordView.swift [hunk 1]
```diff
@@ -128,6 +140,7 @@ public struct ResetPasswordView: View {
                         .shadowCardStyle(bgColor: Theme.Colors.accentColor,
                                          textColor: Theme.Colors.white)
                         .padding(.top, 80)
+                        .accessibilityIdentifier("showalert_text")
```
Comment (shafqat-muneer) [inline Authorization/Authorization/Presentation/Reset Password/ResetPasswordView.swift:143]: Suggestion to replace name with `show_alert_text`

File: Authorization/Authorization/Presentation/Reset Password/ResetPasswordView.swift [hunk 2]
```diff
@@ -144,6 +157,7 @@ public struct ResetPasswordView: View {
                 VStack {
                     Spacer()
                     SnackBarView(message: viewModel.errorMessage)
+                        .accessibilityIdentifier("showerror_snackbar")
```
Comment (shafqat-muneer) [inline Authorization/Authorization/Presentation/Reset Password/ResetPasswordView.swift:160]: Suggestion to replace identifier with `show_error_snackbar`

File: Authorization/Authorization/Presentation/Startup/StartupView.swift [hunk 3]
```diff
@@ -85,6 +89,7 @@ public struct StartupView: View {
                                 .font(Theme.Fonts.bodyLarge)
                         }
                         .padding(.top, isHorizontal ? 0 : 5)
+                        .accessibilityIdentifier("explorer_courses_button")
```
Comment (shafqat-muneer) [inline Authorization/Authorization/Presentation/Startup/StartupView.swift:92]: Suggestion to replace identifier with `explore_courses_button`

File: Core/Core/View/Base/HTMLFormattedText.swift [hunk 4]
```diff
@@ -37,6 +37,7 @@ public struct HTMLFormattedText: UIViewRepresentable {
         textView.textContainerInset = .zero
         textView.textContainer.lineFragmentPadding = 0
         textView.delegate = context.coordinator
+        textView.accessibilityIdentifier = "honer_code_textarea"
```
Comment (shafqat-muneer) [inline Core/Core/View/Base/HTMLFormattedText.swift:40]: Suggestion to replace identifier with `honer_code_text_area`

File: WhatsNew/WhatsNew/Presentation/WhatsNewView.swift [hunk 5]
```diff
@@ -41,6 +41,7 @@ public struct WhatsNewView: View {
                                         .scaledToFit()
                                         .frame(minWidth: 250, maxWidth: 300)
                                         .padding(24)
+                                        .accessibilityIdentifier("whatsnew_image")
```
Comment (shafqat-muneer) [inline WhatsNew/WhatsNew/Presentation/WhatsNewView.swift:44]: Suggestion to replace with `whats_new_image`

File: WhatsNew/WhatsNew/Presentation/WhatsNewView.swift [hunk 6]
```diff
@@ -124,6 +130,7 @@ public struct WhatsNewView: View {
                         .allowsHitTesting(false)
                         .padding(.top, isHorizontal ? 0 : 170)
                         .padding(.bottom, 8)
+                        .accessibilityIdentifier("whatsnew_pagecontrol")
```
Comment (shafqat-muneer) [inline WhatsNew/WhatsNew/Presentation/WhatsNewView.swift:133]: Suggestion to replace with `whats_new_page_control`

**Review by shafqat-muneer — CHANGES_REQUESTED** [review]

File: Core/Core/View/Base/HTMLFormattedText.swift (hunk 4 above)
Comment (saeedbashir) [reply Core/Core/View/Base/HTMLFormattedText.swift:40]: `textarea` is used for field type like textfield.

File: WhatsNew/WhatsNew/Presentation/WhatsNewView.swift (hunk 5 above)
Comment (saeedbashir) [reply WhatsNew/WhatsNew/Presentation/WhatsNewView.swift:44]: whatsnew is actually a feature name and pronounced as single entity.

File: WhatsNew/WhatsNew/Presentation/WhatsNewView.swift (hunk 6 above)
Comment (saeedbashir) [reply WhatsNew/WhatsNew/Presentation/WhatsNewView.swift:133]: Here `pagecontrol` is single entity.

**Review by volodymyr-chekyrta — APPROVED** [review]

**Pull request merged**